cmd/web/config/config_test.go

```diff
@@ -471,3 +489,66 @@ func TestValidatorTypes_AsStringSlice(t *testing.T) {
 		}
 	})
 }
+
+func TestConfig_GetSensored(t *testing.T) {
+
+	const mask = "**masked**"
+
+	cfg := Config{}
+	cfg.Backend.URL = mask
```

Comment:
In line with the previous comment, these should have a different value than `mask`, as you want to verify that the values are actually _masked_ from their original value.

You'll also want to verify this in the test, by doing something like:

```go
gotUnmasked := tt.c.Backend
gotMasked := tt.c.GetSensored().Backend
if gotUnmasked == gotMasked {
// Whelp, testing the wrong thing here
}

if gotMasked != valueMask {
// Whelp, a sensitive value didn't get masked
}
```